inst/schema/scenarioCost.json

```diff
@@ -0,0 +1,14 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "properties": {
+        "id": { "type": "string" },
+        "value": { "type": "number" },
+        "children": {
+            "type": "array",
+            "items": { "$ref": "#" }
```

Comment:
Self-reference, allows the cost to include child components with the same schema.

Reply:
Nice, I didn't know one could do this.